Gems/WhiteBox/Code/Source/Core/WhiteBoxToolApi.cpp

```diff
@@ -2975,10 +2974,8 @@ namespace WhiteBox
             using ModifiedFaceHandle = AZStd::pair<Mesh::FaceHandle, Mesh::FaceHandle>;
             using ModifiedFaceHandles = AZStd::vector<ModifiedFaceHandle>;
 
-            // find all face handles that no longer match (where garbage_collect has invalidated the handles)
-            const ModifiedFaceHandles modifiedFaceHandles = std::transform_reduce(
-                faceHandlesCopy.cbegin(), faceHandlesCopy.cend(), faceHandlePtrs.cbegin(), ModifiedFaceHandles{},
-                // reduce
```

Comment:
suggestion: Sorry I just noticed this but can we add the transform and reduce comments back please, I think they're helpful to know what each lambda is doi